darwin.c

```diff
@@ -7,82 +7,8 @@
 #include <mach-o/dyld.h>
 #include "common.h"
 
-int elevate(char *path, char *prompt, char *iconPath)
```

Comment:
The other benefit to using cocoasudo+elevate is that we can reuse it for other programs when we need it (for example, if we ever need to install certificates).